### ISSUE TITLE
unable to view dashboards in jupyter notebook

### ISSUE THREAD

**Issue by dalavayi** [evidentlyai/evidently#45]
hi, i am trying out the notebooks in examples, but unable to view the dashboards in notebooks, i just stops with a message "Loading.."
PFA the screenshot of the issue
![image](https://user-images.githubusercontent.com/72725647/128799506-93c334bb-0a76-4dc9-9779-6e998baf00b5.png)

**Comment by emeli-dral** [comment]
Hi @dalavayi !

Could you please tell, if the report saving works well (command ```save('path')``` instead ```show()```)? It will help to figure out the reason of the error.

And just to check: make sure you had nbextention installed and enables before using the tool to generate reports in the notebook:
```$ jupyter nbextension install --sys-prefix --symlink --overwrite --py evidently```
```$ jupyter nbextension enable evidently --py --sys-prefix```

**Comment by dalavayi** [comment]
hi @emeli-dral , the .save() method is working. 
I tried running the commands shared above, but unable to install the nbextension due to this error :
OSError: [WinError 1314] A required privilege is not held by the client: 'mypath\\lib\\site-packages\\evidently\\nbextension\\static' -> 'myapath\\share\\jupyter\\nbextensions\\evidently'

**Comment by rmminusrslash** [comment]
I am having the same issue. If I reduce the dataset to one feature, the report shows. If I use 64 features and 150K datapoints, the report is goes from 16MB to 600 MB and not displaying (saved or in jupyter).

**Comment by emeli-dral** [comment]
@dalavayi 

We tried to reproduce the error on our side and we believe that the problem may be in the absence of the rights to create a symlink. We tried to install the nbextension without using a symlink and it worked for us. 

I suggest you do the same, meaning instead of the command, try to fill in the command

**Comment by emeli-dral** [comment]
Hi @dalavayi,

I wanted to let you know that we updated the ```show()``` method. Now it has the argument ```mode```, which can take the following options:
- auto - the default option. Ideally, you will not need to specify the value for mode and use the default. But, if it does not work (in case we failed to determine the environment automatically), consider setting the correct value explicitly.
- nbextension - to show the UI using nbextension. 
- inline - to insert the UI directly into the cell.

I suggest you to try run the ```show``` method with the``` mode='inline'```. This might help to load visuals without the need to install nbextention.

Also, ```evidently``` now works on Google Colab, you might find it convenient as well. Here is an example: [Iris](https://colab.research.google.com/drive/1uFN0ZWNLk3ExylBywq_M8oGkmac0lDrq)
